graphene_federation/main.py

```diff
@@ -20,9 +20,11 @@ def build_schema(query, **kwargs):
     return graphene.Schema(query=_get_query(schema, query), **kwargs)
 
 
-def key(fields: str):
+def key(fields: str, *args: str):
     def decorator(Type):
         register_entity(Type.__name__, Type)
-        setattr(Type, '_sdl', '@key(fields: "%s")' % fields)
+        sdl = '@key(fields: "%s")' % fields
```

Comment:
something like this might be easier to read (if this is what you're going for )

`sdl = f'@key(fields: "{ fields}")'`